test/unit/oidc_jwt_token_validation_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'ostruct'
+
+class OidcJwtTokenValidationTest < ActiveSupport::TestCase
```

Comment:
This should be `OidcJwtTokenValidateTest`. Please also update the filename accordingly.